Modules/_zstd/compressor.c

```diff
@@ -26,8 +26,8 @@ class _zstd.ZstdCompressor "ZstdCompressor *" "clinic_state()->ZstdCompressor_ty
 #define ZstdCompressor_CAST(op) ((ZstdCompressor *)op)
 
 int
```

Comment:
```suggestion
static int
```

I believe these should have static linkage to keep this scoped to this translation unit.

Reply:
You're right, fixed in [d9cdafb](https://github.com/python/cpython/pull/133535/commits/d9cdafb3c17d6398d1a6f20f1fd7740df3499725)